lib/commons/aria/required-context.js

```diff
@@ -9,15 +9,13 @@ import lookupTable from './lookup-table';
  * @return {Mixed} Either an Array of required context elements or `null` if there are none
  */
 function requiredContext(role) {
-	'use strict';
-	let context = null;
-	const roles = lookupTable.role[role];
+	const roleDef = standards.ariaRoles[role];
 
-	if (roles) {
-		// TODO: es-module-utils.clone
-		context = axe.utils.clone(roles.context);
+	if (!roleDef || !Array.isArray(roleDef.requiredContext)) {
+		return null;
 	}
-	return context;
+
+	return roleDef.requiredContext;
```

Comment:
We should return a copy here. The return value isn't immutable, we can't assume some rule isn't going to accidentally modify this object.